vector-search/vector-search-quickstart.ipynb

```diff
@@ -0,0 +1,703 @@
+{
```

Comment:
<p>This is very good to include!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
Thanks! I myself really needed this.